canonical-web-and-design/ubuntu.com
# More text changes on the robotics page

## Done

- text changes on the robotics page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/robotics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare it to the [copy doc](https://docs.google.com/document/d/1cDxhPCQbHA0r6n9JXNmO50ANLPZhCFdgwATX6f90guU/edit#)

## Issue / Card

Fixes #5669
